tsconfig.json

```diff
@@ -23,6 +23,7 @@
     "noImplicitOverride": true,
   },
   "include": [
-    "src/**/*"
+    "src/**/*",
+    "dev/**/*"
```

Comment:
⚠️ This has impact on the bundles.
This probably reinclude the dev/demo code within the bundles, so the rollup config file needs to be reviewed
In addition we should declare `dev/ts/**/*` instead

https://github.com/process-analytics/bpmn-visualization-js/blob/v0.21.1/rollup.config.js#L125-L129